lib/AST/Expr.cpp

```diff
@@ -277,6 +277,10 @@ bool Expr::isSemanticallyConstExpr(
   return printConstExprValue(nullptr, additionalCheck);
 }
 
+bool Expr::isDistributedKnownToBeLocal() const {
+  assert(false && "fixme: should know the attr");
```

Comment:
I assume you'll be looking for a `DeclRefExpr` storing a `_local` parameter?

Reply:
Yes, I'll we'll dig through the expr to find the `base` of the Call or property access, and if it has a `_local` we say `true`.

I always have trouble figuring out how to find the base, but I think we've done this elsewhere before.